src/admin-views/aggregator/status.php

```diff
@@ -151,16 +187,14 @@
 		// Meetup status section
 		$indicator = 'good';
 		$notes = '&nbsp;';
-		$text = '&nbsp;';
+		$text = __( 'API key entered', 'the-events-calendar' );
 		$meetup_api_key = tribe_get_option( 'meetup_api_key' );
 		if ( ! $meetup_api_key ) {
 			$indicator = 'warning';
-			$text = __( 'You have not set your Meetup API key.', 'the-events-calendar' );
-			$notes = sprintf( // add link to API tab
-				__( 'Add your API key on the %1$sSettings &gt; APIs%2$s page', 'the-events-calendar' ),
-				'<a href="' . esc_url( Tribe__Settings::instance()->get_url( array( 'tab' => 'addons' ) ) ) . '">',
-				'</a>'
-			);
+			$text = __( 'You have not entered a Meetup API key', 'the-events-calendar' );
+			$notes = '<a href="' . esc_url( Tribe__Settings::instance()->get_url( array( 'tab' => 'addons' ) ) ) . '">';
+			$notes .= __( 'Enter your API key', 'the-events-calendar' );
```

Comment:
Similar to one of the changes at the start, we could make this a little more readable and remove `$notes` from each subsequent line by doing `. __( ... )` on each line